.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+Description of work.
+
+**To test:** Instructions for testing.
+
+Fixes #xxxx.
+
+Either link to [Release notes](http://www.mantidproject.org/Release_Notes_3.7) change-set or state "Does not need to be in the release notes."
```

Comment:
@MichaelWedel We discussed that at some point last year but there was a concern over frequent merge conflicts on editing the same files. I think it would be better if they were version with the code too as then if a PR isn't merged, say for close to an official release, then the release notes wouldn't appear and we wouldn't forget to remove them.